spec/features/applications/create_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+RSpec.describe "User Story 2: Application New Page" do
+  describe "As a visitor, when I visit the pet index page" do   
+    before(:each) do
+      @shelter = Shelter.create(name: "Aurora shelter", city: "Aurora, CO", foster_program: false, rank: 9)
+
+      @pet_1 = Pet.create(adoptable: true, age: 7, breed: "sphynx", name: "Bare-y Manilow", shelter_id: @shelter.id)
+      @pet_2 = Pet.create(adoptable: true, age: 3, breed: "domestic pig", name: "Babe", shelter_id: @shelter.id)
+      @pet_3 = Pet.create(adoptable: true, age: 4, breed: "chihuahua", name: "Elle", shelter_id: @shelter.id)
+    end
+
+    context "When I visit the pet index page" do
+      it "I see a link to Start an Application, and I click this link, then I am taken to the new application page where I see a form" do
+
+        visit "/pets"
+        expect(page).to have_link("Start an Application")
+        click_link("Start an Application")
+        expect(current_path).to eq("/applications/new")
+
```

Comment:
This test from line 14 to line 19 could also live in your `pets/index_spec.rb` to test that the link lives on that page.